packages/ubuntu_desktop_installer/lib/widgets/mascot_avatar.dart

```diff
@@ -1,24 +1,28 @@
 import 'package:flutter/material.dart';
 
+const _kMascotAsset = AssetImage('assets/mascot.png');
+
 class MascotAvatar extends StatelessWidget {
-  const MascotAvatar({super.key, this.radius = 80});
+  const MascotAvatar(
+      {super.key, this.image = _kMascotAsset, this.height = 160});
+
+  final ImageProvider<Object> image;
+  final double height;
```

Comment:
should we either expose `width` or call it `size` instead and pass it as width to make it safe to use this in horizontal layouts too?